include/E/E_RandomDistribution.hpp

```diff
@@ -60,6 +60,7 @@ class ExpDistribution : public RandomDistribution
 class LinearDistribution : public RandomDistribution
 {
 public:
+	LinearDistribution();
```

Comment:
This is automatically generated by the compiler. As I know, this is not necessary.